classes/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Import blocks
+from . import blocks
+
+# Import files
+from . import BasicClasses
```

Comment:
This structure does not seem to run.
Doing something like:
<pre>
from .BasicClasses import Chunk, Block, Region
from .WorldGenerator import WorldGenerator, GENERATORS
</pre>
Does however run. Maybe it's just on my side (although I wonder where I went wrong then), but if it's not, then it's an easy fix

Reply:
__init__ is used here to load files, I don't think import like your suggestion is good, we don't have to import classes but only load python files.
I'll check if files are loaded when I'll be on my computer

Reply:
Well, it's sorted now, looks as if it works, what went wrong, idk, but the issue was on my side